Poisson_surface_reconstruction_3/include/CGAL/Reconstruction_triangulation_3.h

```diff
@@ -348,12 +348,12 @@ class Reconstruction_triangulation_3
   /// Insert the [first, beyond) range of points in the triangulation using a spatial sort.
   /// Default type is INPUT.
   ///
-  /// @commentheading Template Parameters:
-  /// @param InputIterator iterator over input points.
-  /// @param PointPMap is a model of `ReadablePropertyMap` with a value_type = Point_3.
+  /// *Template Parameters:*
```

Comment:
Is this line not a bit overdone as doxygen will add a header **Template Parameters**